scripts/fix_lotus_encumbrances/README.md

```diff
@@ -1,6 +1,6 @@
 # Fix Lotus Encumbrances
 
-This script will fix several issues in encumbrances related to the fiscal year rollover for FOLIO Lotus.
+This script will fix several issues in encumbrances related to the fiscal year rollover for FOLIO Lotus, and some other encumbrance issues still happening with later releases.
```

Comment:
Could it be more specific? I beleive this story is about negative amounts issue

Reply:
I tried to be more specific - see commit.